README.md

```diff
@@ -1 +1,227 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+<details>
+  <summary>State Management</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Redux</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Fast data fetching and Caching]**
+- **[Intuitive UI]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://63ce65ee6de26f3b068bbbd6--gregarious-croissant-eb0f67.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+- [Video-presentation](https://drive.google.com/file/d/1EdGAYtKE1KW9DTjJrs8zkQjjaf1bwrR8/view?usp=sharing)
+
+## 🚀 Video Presentation <a name="live-demo"></a>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+```
+  git clone https://github.com/Victoria129/Air-Pollution.git
+```
+
+### Prerequisites
+
+In order to run this project you need:
+
+``` node
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/Victoria129/Air-Pollution.git
+```
+
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd my-project
+  npm install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```sh
+  npm run dev
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+  npm run test
+```
+
+
+### Deployment
+
+You can deploy this project using:
+
+
+```npm run dev
+
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Moses Wamae**
+
+- GitHub: [@Victoria129](https://github.com/Victoria129)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/victoria-tumwebaze/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Loading Screens]**
+- [ ] **[Further Details on Air pollution]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Victoria129/Air-Pollution/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```

Comment:
The Acknowledgment section is empty, I think you should mention the author of the original design here.

![Screenshot from 2023-03-17 17-09-26](https://user-images.githubusercontent.com/17243325/225930421-bb58e0e5-3b4c-4da8-87ae-327b9d122f11.png)